contributing/community/review-comments.rst

```diff
@@ -149,6 +149,17 @@ you don't have to use "Please" all the time. But it wouldn't hurt.
 It may not seem like much, but saying "Thank you" does make others feel
 more welcome.
 
+
+Preventing Escalations
+------------------
```

Comment:
The title underline should be exactly the same length as the title itself to make restructured text happy